.gitignore

```diff
@@ -1,2 +1,10 @@
 build/
 **/.DS_Store
+
+# iOS Example directories (large circuit files)
+ios/ExampleApp/IMP1FrameworkExample/Examples/100k/
+ios/ExampleApp/IMP1FrameworkExample/Examples/200k/
+ios/ExampleApp/IMP1FrameworkExample/Examples/400k/
+ios/ExampleApp/IMP1FrameworkExample/Examples/800k/
+ios/ExampleApp/IMP1FrameworkExample/Examples/1600k/
+ios/ExampleApp/IMP1FrameworkExample/Examples/rarimo-bionet/
```

Comment:
It doesn't look like these files were ignored. Make sure to remove them from your git cache to make them ignored